### PR TITLE
Add missed description for UG sorting

### PR DESCRIPTION
**Pull request by Echomo-Xinyu**

### CODE REVIEW

**Review by sjoann — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -574,7 +574,8 @@ Format: `sort TYPE [ORDER]`
   - `DESC`
   - If it is left as blank, it will follow the default order based on the specified`TYPE`.
```
Comment (gnahzeus) [inline docs/UserGuide.md:575]: Seems to have a missing space in this line? specifiedTYPE

**Review by gnahzeus — APPROVED** [review]
LGTM, spotted a minor spacing issue.

File: docs/UserGuide.md (hunk 1 above)
Comment (Echomo-Xinyu) [reply docs/UserGuide.md:575]: Noted! Good catch haha

**Pull request merged**